Makefile

```diff
@@ -93,6 +93,11 @@ go.test.unit: $(KUBEBUILDER)
 manifests: vendor kubebuilder.manifests
 	@$(INFO) Generating CRD manifests
 	$(CONTROLLERGEN) crd:maxDescLen=0,trivialVersions=true paths=./apis/stacks/... output:dir=$(CRD_DIR)
+# Add "helm.sh/hook: crd-install" and "helm.sh/hook-delete-policy: before-hook-creation" annotations for clusterstackinstalls and stackinstalls CRDs
```

Comment:
do you think another quick comment about why `crd-install` is the right hook, and what specifically it does for us (why is it important) would be useful here?

Reply:
Done